### DISCUSSION TITLE
Quiz 2, Data Visualization Specialization

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#693] [Q&A] [answered]
Mau tanya. Untuk nomor 5, jawaban hasil analisis saya kok gaada di opsi jawabannya ya? Apa ada kesalahan di code saya?

Berikut repo github untuk quiz saya:
https://github.com/ArdiantheGreat/Algoritma_School

**Comment by kinantytasyao** [comment]
Dear @ArdiantheGreat ,

Hal tersebut terjadi karena data yang digunakan seharusnya adalah data kiva hanya untuk negara Philippines.

Demikian yang dapat kami sampaikan. Jika membantu, boleh klik mark as answer. Terima kasih :)

**Reply by naturallyfunny** [reply, marked as answer]
Ternyata saya keliru di codingannya, funding durationnya saya difftime posted_time sama **funded_amount** :)
